MSAL/src/MSALErrorConverter.m

```diff
@@ -62,6 +62,7 @@ + (void)initialize
                                    @(MSIDErrorInteractiveSessionStartFailure) : @(MSALErrorInternal),
                                    @(MSIDErrorInteractiveSessionAlreadyRunning) : @(MSALErrorInteractiveSessionAlreadyRunning),
                                    @(MSIDErrorNoMainViewController) : @(MSALErrorNoViewController),
+                                   @(MSIDErrorServerUnhandledResponse): @(MSALErrorUnhandledResponse)
```

Comment:
This might not be needed anymore after you merge your Error Converter PR :)

Reply:
yes, step by step :)